edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Startup.cs

```diff
@@ -221,5 +227,62 @@ string GetStoragePath()
             var value = this.Configuration.GetValue<T>(key);
             return EqualityComparer<T>.Default.Equals(value, default(T)) ? Option.None<T>() : Option.Some(value);
         }
+
+        Option<IMetricsRoot> BuildMetricsCollector()
+        {
+            Option<IMetricsRoot> metricsCollector = Option.None<IMetricsRoot>();
+            bool collectMetrics = this.Configuration.GetValue("CollectMetrics", false);
+            if (!collectMetrics)
+            {
+                return metricsCollector;
+            }
+
+            string metricsStoreType = this.Configuration.GetValue<string>("MetricsStoreType");
+            
+            if (metricsStoreType == "influxdb")
+            {
+                string metricsDbName = this.Configuration.GetValue("MetricsDbName", "metricsdatabase");
+                string influxDbUrl = this.Configuration.GetValue("InfluxDbUrl", "http://127.0.0.1:8086");
```

Comment:
This URL may not be the right default value in Edge scenarios..

Reply:
I mean when EdgeHub is running in one container and InfluxDb is running in another container.

Reply:
If the influx container is on the same host as the edge hub container, this will work. If the influx container is on a different host, then the user will have to supply the ip address of that host.

Reply:
Emm, how will it work even on the same host? Won't 127.0.0.1 resolve to the EdgeHub container itself?

Reply:
Yes, you're right. I'll change it to a better default.